cookbooks/bcpc-hadoop/templates/default/ooz_oozie-env.sh.erb

```diff
@@ -9,19 +9,21 @@
 <% if @ooz_hosts.length >= 2 %>
 export OOZIE_HTTP_HOSTNAME=<%=node[:bcpc][:management][:vip]%>
 <% end %>
-export OOZIE_CONFIG="/etc/oozie/conf"
-export OOZIE_DATA="/var/lib/oozie"
-export OOZIE_LOG="/var/log/oozie"
-export OOZIE_LOG_DIR="/var/log/oozie"
-export OOZIE_PID_DIR="/var/run/oozie"
-export OOZIE_CATALINA_HOME="/usr/lib/bigtop-tomcat"
-export CATALINA_TMPDIR="/tmp"
-export CATALINA_PID="/var/run/oozie/oozie.pid"
-export CATALINA_BASE="/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/oozie/oozie-server/"
+export OOZIE_CONFIG=/etc/oozie/conf
+export OOZIE_DATA=/var/lib/oozie
+export OOZIE_LOG=/var/log/oozie
+export OOZIE_LOG_DIR=/var/log/oozie
+export OOZIE_PID_DIR=/var/run/oozie
+export OOZIE_CATALINA_HOME=/usr/lib/bigtop-tomcat
+export CATALINA_TMPDIR=/tmp
+export CATALINA_PID=/var/run/oozie/oozie.pid
+export CATALINA_BASE=/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/oozie/oozie-server/
 export CATALINA_OPTS="<%= node["bcpc"]["hadoop"]["oozie"]["memory_opts"] %>"
 export OOZIE_HTTPS_PORT="11443"
 export OOZIE_HTTPS_KEYSTORE_PASS="<%=get_config("oozie-keystore-password") %>"
 export CATALINA_OPTS="$CATALINA_OPTS -Doozie.https.port=${OOZIE_HTTPS_PORT}"
 export CATALINA_OPTS="$CATALINA_OPTS -Doozie.https.keystore.pass=${OOZIE_HTTPS_KEYSTORE_PASS}"
-export JAVA_HOME="<%= node["bcpc"]["hadoop"]["java"] %>"
-export JAVA_LIBRARY_PATH="/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/hadoop/lib/native:/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/hadoop/lib/native/Linux-amd64-64"
+export JAVA_HOME=<%= node["bcpc"]["hadoop"]["java"] %>
+export JAVA_LIBRARY_PATH=/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/hadoop/lib/native:/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/hadoop/lib/native/Linux-amd64-64
+export OOZIE_HTTP_HOSTNAME=<%= float_host(node[:fqdn]) %>
+export OOZIE_BASE_URL=http://<%= float_host(node[:bcpc][:management][:viphost]) %>:11000/oozie
```

Comment:
The quotes were removed just to match `Oozie` [documentation](https://oozie.apache.org/docs/4.0.1/AG_Install.html).